index.html

```diff
@@ -1,22 +1,67 @@
 <!DOCTYPE html>
 <html>
+	<head>
+	    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/css/bootstrap.min.css" integrity="sha384-PsH8R72JQ3SOdhVi3uxftmaW6Vc51MKb0q5P2rRUpPvrszuE4W1povHYgTpBfshb" crossorigin="anonymous">
+	</head>
 	<body>
-		<h1>My First Heading</h1>
-		<p>My first paragraph.</p>
-
-		<div id="target"></div>
+		<div class= "container">
+		  <div class="row">
+			  <div class="col">
+				<div id="target"></div>
+			  </div>
+		  </div>		   
+		</div>
 
+			
 		<script src="https://cdn.jsdelivr.net/npm/ractive"></script>
 
 		<script id="template" type="text/ractive">
-			<p>{{greeting}}, {{name}}!</p>
+			<h1>{{title}}</h1>
+			<h5>{{subtitle}}</h5>
+
+			<table class="table">
+				<tr>
+				  <th>Name</th>
+				  <th>Age</th>
+				  <th>Is Minor</th>
+				  <th>Actions</th>
+				</tr>
+				{{#each people}}
+				  <tr>
+					<td>{{name}}</td>
+					<td>{{age}}</td>
+					<td>{{isMinor}}</td>
```

Comment:
One bug is that if the age were changed it wouldn't update the isMinor.  You could solve this by just calculating the column.   {{age < 21}}